donkeycar/management/graph.py

```diff
@@ -100,8 +100,10 @@ def get_x(self, num_points):
         return x_trafo
 
     def transform_y(self, y):
-        y_scale = (self.bounding_box[1][1] - self.bounding_box[0][1]) \
-                  / (y.max() - y.min())
+        denom = (y.max() - y.min())
```

Comment:
Just add a very small epsilon value (something like `1e-6` so you don't have have a divide by zero case.